code_generator/End2End/GeneratedDependencies/before-compile.sh

```diff
@@ -0,0 +1,6 @@
+ROOT=`dirname $0`
```

Comment:
I'd just embed these directly into the project file as build commands, rather than relying on bash.

Reply:
Done